mage/main_test.go

```diff
@@ -75,6 +75,21 @@ func TestVerbose(t *testing.T) {
 	}
 }
 
+func TestGoRunList(t *testing.T) {
+	c := exec.Command("go", "run", "main.go")
+	c.Dir = "./testdata/no_default"
+	c.Env = os.Environ()
+	b, err := c.CombinedOutput()
+	if err != nil {
+		t.Error("error:", err)
+	}
+	actual := string(b)
+	expected := "  bazBuz    Prints out 'BazBuz'.\n  fooBar    Prints out 'FooBar'.\n"
```

Comment:
this is nitpicky, but can you make this a multiline string, i.e. use \`\` instead?  I think it'll read a lot easier.  A trick I use is to use a line return at the beginning of the string and then immediately slice it off, to make it easy to read, e.g.  

```
    expected := `
  bazBuz     Prints out 'BazBuz'.
  fooBar .   Prints out 'FooBar'.
`[1:]

The slice then removes that initial line return, but it keeps the string all nicely left adjusted.

Reply:
Perfect. Will do.